docs/catalog/web/minimize-main-thread-work.md

```diff
@@ -0,0 +1,42 @@
+---
+version: 1.0
+submitted_by: markus-ntt-seidl
+published_date: TBD
+category: web
+tags: 
+ - web
+ - role:web-developer
+ - size:small
+---
+
+# Minimize main thread work
+
+## Description
+
+Web browsers traditionally consist of a main rendering thread that handles most of the updates on the web page and the execution of JavaScript. JavaScript is executed on the main thread to simplify JavaScript implementations so the JavaScript programmer doesn't have to deal with any multi-threading programming patterns.
+When doing long running JavaScript computations it is running single threaded. In contrast all CPUs nowadays have multiple cores which have to be powered on while being underutilized.
+
+## Solution
+
+For long running JavaScript computations try to use WebWorkers and run them in another thread while keeping the main rendering thread free. Consider moving the JavaScript computation to efficient server implementations that use optimized algorithms.
+
+## SCI Impact
+
+`SCI = (E * I) + M per R`  
+[Software Carbon Intensity Spec](https://grnsft.org/sci)
+
+Concerning the SCI equation, minimize main thread work will impact as follows:
+
+- `I`: CPU resources can be used more efficiently which reduces the electricity required and reduces the carbon intensity
```

Comment:
I wonder if this should be an `E` instead of `I`. It can reduce the overall energy by parallelizing the work but doesn't seem to impact the intensity itself.